src/io/file.js

```diff
@@ -2,7 +2,41 @@
 // http://github.com/p-v-o-s/infragram-js.
 
 module.exports = function File(options, processor) {
-
+  let getMouseEvent = () => {
```

Comment:
Is this stuff from merging from the main branch? I think it's related to https://github.com/publiclab/infragram/pull/324, no? Ideally if you're not changing any files in `/src/` or `/dist/` you shouldn't see them in your PR, so maybe some got pulled in somehow. This could be related to your conflicts. 

Sometimes if resolving conflicts is too complex and if I don't want entire files changed, I just copy the raw content of the file from the `main` branch, and paste it manually into the file in my local working branch. Then if I commit those changes, it shows no difference from the main branch and the files disappear from my pull request. Does that make sense? I'm also happy to try resolving these if you'd like me to take a crack at them!